app/views/sessions/_card.html.erb

```diff
@@ -1,35 +1,62 @@
-<div class="w-full p-5 bg-white rounded-lg">
-    <div class="flex flex-row items-center mb-6">
-      <% if speaker.profile.image.attached? %>
-        <%= image_tag url_for(profile.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
-      <% else %>
-        <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
-            <%= image_tag url_for("user_avatar.svg") %>
+<div class="flex flex-col w-full p-5 mb-4 bg-white rounded-[20px]">
+  <div class="flex flex-col w-full">
+    <div class="flex justify-between w-full mb-4">
+      <div class="flex">
+        <% if speaker.image.attached? %>
+          <%= image_tag url_for(speaker.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
+        <% else %>
+          <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
+            <%= inline_svg_tag "icons/user_avatar.svg" %>
+          </div>
+        <% end %>
+
+        <div class="flex flex-col justify-center h-full">
+          <a href="<%= speaker_path(speaker) %>">
+            <h2 class="italic font-black text-red"><%= speaker.name %></h2>
+            <p class="text-sm text-gray-800"><%= speaker.job_title %></p>
+          </a>
         </div>
-      <% end %>
-      <div class="flex flex-row items-center justify-between w-full grow">
-        <div class="flex flex-col">
-          <p class="text-base italic font-black text-red"><%= speaker.profile.name %></p>
-          <p class="font-normal text-[13px]"><%= speaker.profile.job_title %></p>
-        </div>
-        <%= inline_svg_tag "circle_with_checkmark.svg", class: "fill-red-500" %>
       </div>
-    </div>
-    <h1 class="mb-4 text-xl italic font-bold text-red"><%= session.title %></h1>
-    <div class="flex flex-row flex-wrap items-center gap-2 mb-[18px]">
-    <% session.tags.each do |tag| %>
-      <div class="bg-white border border-blue-2 rounded-lg px-4 py-[6px] text-blue-2">
-      <%= tag.name %>
+
+      <div class="flex items-center px-1">
+        <%= button_to session_path(id: session.id, on_date: request.params[:on_date]), method: :put, form_class: "flex items-center" do %>
+          <% if current_user.sessions.include?(session) %>
```

Comment:
I don't think we need to be worrying about n+1 given the expected DB size, however, if time allows, consider having a single query in sessions/index and then sending a local to the partial that determines if the icon needs to be displayed or not.